src/App.tsx

```diff
@@ -15,71 +15,79 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export const App: React.FC = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const goodsList = goods.map(good => (
+    <tr
+      className={
+        (selectedGood === good)
+          ? ('has-background-success-light')
+          : ''
+      }
+      data-cy="Good"
+    >
+      <td>
+        {
+          (selectedGood !== good)
+            ? (
```

Comment:
start with positive if you're working with booleans. Positive booleans are much easier to read
```suggestion
        {selectedGood === good
            ? (
```